subsquid/src/dbHelper.ts

```diff
@@ -1,43 +1,44 @@
-import {Store} from "@subsquid/substrate-processor";
-import {PabloPool} from "./model";
+import { Store } from "@subsquid/substrate-processor";
+import { PabloPool } from "./model";
 
 export async function get<T extends { id: string }>(
-    store: Store,
-    EntityConstructor: EntityConstructor<T>,
-    id: string
+  store: Store,
+  EntityConstructor: EntityConstructor<T>,
+  id: string
 ): Promise<T | undefined> {
-    return await store.get<T>(EntityConstructor, {
-        where: {id},
-    });
+  return await store.get<T>(EntityConstructor, {
+    where: { id },
+  });
 }
 
 export async function getLatestPoolByPoolId<T extends { id: string }>(
-    store: Store,
-    poolId: bigint
+  store: Store,
+  poolId: bigint
 ): Promise<PabloPool | undefined> {
-    return await store.get<PabloPool>(PabloPool, {
-        where: {poolId},
-        order: {calculatedTimestamp: 'DESC'},
-    });
+  return await store.get<PabloPool>(PabloPool, {
+    where: { poolId },
+    order: { calculatedTimestamp: "DESC" },
+    relations: ["poolAssets"],
```

Comment:
I'm not sure if this is the right solution, as this requires any function that calls the db to explicitly state the which ones to load eagerly.